_posts/2018-09-08-terminal-tips.md

```diff
@@ -4,3 +4,26 @@ tags: linux, bash, compilation
 ---
 
 // Start your contributions here
+*Contributed by: <your name>*
+### Title for the tip/code
+    insert code here (if any)
+description about what your tip does with accompanying code snippet
+
+---
+
+*Contributed by: Bhargav SNV*
+### Stop/Start Displaying Terminal Input
+    stty -echo
+    stty echo
+When `stty -echo` is done, input in the terminal won't be displayed.
+When `stty echo` is done, input in the terminal will be displayed.
+
+### Describe Filetype of File or Directory
+    file [file/dir]
+The command describes the type of file/directory provided as a n argument along with a description of its contents.
+
+### Create a typescript of terminal session
```

Comment:
Add an example with output here also, for extra clarity

Reply:
@tfidfwastaken, I can't really do so for this, but I surely can for the previous ones!